backend/de-metas-salesorder/src/main/java/de/metas/salesorder/service/AutoProcessingOrderService.java

```diff
@@ -118,6 +118,10 @@ public void completeShipAndInvoice(@NonNull final OrderId orderId)
 		final GenerateShipmentsForSchedulesRequest request = GenerateShipmentsForSchedulesRequest.builder()
 				.scheduleIds(scheduleIds)
 				.quantityTypeToUse(M_ShipmentSchedule_QuantityTypeToUse.TYPE_QTY_TO_DELIVER)
+				
+				// Usually we want only *CUs* to be picked on-the-fly, because we don't know or care which HUs are shipped and don't want to make assumptions regarding their TU-Packaging.
```

Comment:
```suggestion
				// Usually we want only *CUs* to be picked on-the-fly, because we don't know and care which HUs are shipped and don't want to make assumptions regarding their TU-Packaging.
```